lib/logstash/outputs/neo4j.rb

```diff
@@ -0,0 +1,35 @@
+require 'logstash/namespace'
+require 'logstash/outputs/base'
+require 'logstash/outputs/timetree/timetree'
```

Comment:
I don't like the idea of using a LS namespace as this is at the end going/or intended to be part of a client gem. Is something that at the end should be logstash agnostic.